src/plugins/intel_cpu/tests/unit/shape_inference_test/squeeze_shape_inference_test.cpp

```diff
@@ -16,14 +16,126 @@ using namespace ov;
 using namespace ov::intel_cpu;
 using namespace testing;
 
-class SqueezeStaticShapeInferenceAssertTest : public OpStaticShapeInferenceTest<op::v0::Squeeze> {
+namespace v0 {
+
+class SqueezeV0StaticShapeInferenceAssertTest : public OpStaticShapeInferenceTest<op::v0::Squeeze> {
+protected:
+    void SetUp() override {
+        output_shapes = ShapeVector(1);
+    }
+};
+
+TEST_F(SqueezeV0StaticShapeInferenceAssertTest, no_axes) {
+    const auto arg = std::make_shared<op::v0::Parameter>(element::f64, PartialShape{-1, -1});
+    const auto axes = std::make_shared<op::v0::Parameter>(element::i64, PartialShape{1});
+    const auto op = make_op(arg, axes);
+
+    input_shapes = ShapeVector{{5, 6}, axes->get_shape()};
+
+    OV_EXPECT_THROW(shape_inference(op.get(), input_shapes),
+                    NodeValidationFailure,
+                    HasSubstr("Check 'constant != nullptr'"));
+}
+
+TEST_F(SqueezeV0StaticShapeInferenceAssertTest, parameter_static_shape_axes_no_data) {
+    const auto arg = std::make_shared<op::v0::Parameter>(element::f64, Shape{2, 1, 3, 1});
+    const auto axes = std::make_shared<op::v0::Parameter>(element::i64, Shape{2});
```

Comment:
```suggestion
    const auto arg = std::make_shared<op::v0::Parameter>(element::f64, ov::PartialShape::dynamic());
    const auto axes = std::make_shared<op::v0::Parameter>(element::i64, [Shape{2}](ov::PartialShape::dynamic()));
```
There is no need to set static shape here

Reply:
Undo, the change cause test fail:
`
[ RUN      ] SqueezeV0StaticShapeInferenceAssertTest.no_axes
unknown file: Failure
C++ exception with description "Exception from src/core/src/partial_shape.cpp:266:
to_shape was called on a dynamic shape.
" thrown in the test body.
[  FAILED  ] SqueezeV0StaticShapeInferenceAssertTest.no_axes (0 ms)
`